ra_aid/proc/interactive.py

```diff
@@ -57,7 +57,7 @@ def cleanup():
         os.environ['PAGER'] = ''
         
         # Run command with script for TTY and output capture
-        os.system(f"script -q -c {shlex.quote(shell_cmd)} {shlex.quote(output_path)}")
+        os.system(f"script -q {shlex.quote(output_path)} {shell_cmd}")
```

Comment:
For this one we'll need some kind of OS or script version detection, so it works with both... annoying but should be straightforward.